### ISSUE TITLE
can this do SVG to PNG conversion

### ISSUE THREAD

**Issue by gedw99** [gonum/plot#230]
i need to be able to take a standard SVG, and convert it to PNG or JPG.
Not sure if this can do that ?

**Comment by sbinet** [comment]
hum... no I don't think so.
`plot` can use `github.com/ajstarks/svgo`, generate a SVG plot and let the browser render it.
`plot` can use `image/{png,jpeg}`, generate a `{png,jpeg}` file and let `${IMAGE_VIEWER}` render it.

but it can't go from one to the other.

I suspect `github.com/ajstarks/svgo` may have a way to encode a `png/jpeg` from its internal representation of a `SVG` scene, though.

**Comment by sbinet** [comment]
also, you could either file an issue against svgo or `os/exec.Command` inkscape.

**Comment by gedw99** [comment]
@sbinet 
yeah, i get the impression that svg to raster transcoding is not something SVGO person wants to do.
Regarding inkscape, I dont want something as heavy as inkscape. i know it will work. But i prefer a simple c++ dll wrapper.
